rancher2/resource_rancher2_machine_config_v2.go

```diff
@@ -38,29 +39,51 @@ func resourceRancher2MachineConfigV2Create(d *schema.ResourceData, meta interfac
 
 	d.SetId(newObj.ID)
 	d.Set("kind", newObj.TypeMeta.Kind)
-	stateConf := &resource.StateChangeConf{
-		Pending:    []string{},
-		Target:     []string{"active"},
-		Refresh:    machineConfigV2StateRefreshFunc(meta, newObj.ID, newObj.TypeMeta.Kind),
-		Timeout:    d.Timeout(schema.TimeoutCreate),
-		Delay:      1 * time.Second,
-		MinTimeout: 3 * time.Second,
-	}
-	_, waitErr := stateConf.WaitForState()
-	if waitErr != nil {
-		return fmt.Errorf("[ERROR] waiting for machine config (%s) to be active: %s", newObj.ID, waitErr)
+
+	err = waitForMachineConfigV2State(meta.(*Config), d, meta, d.Timeout(schema.TimeoutCreate))
+	if err != nil {
+		return fmt.Errorf("[ERROR] waiting for machine config (%s) to be active: %s", newObj.ID, err)
 	}
 
 	return flattenMachineConfigV2(d, newObj)
 }
 
+func waitForMachineConfigV2State(c *Config, d *schema.ResourceData, meta interface{}, interval time.Duration) error {
+	log.Printf("[INFO] Waiting for state Machine Config V2 %s", d.Id())
+
+	ctx, cancel := context.WithTimeout(context.Background(), interval)
+	defer cancel()
+	for {
+		kind := d.Get("kind").(string)
+		_, err := getMachineConfigV2ByID(meta.(*Config), d.Id(), kind)
+		if err == nil {
+			return nil
+		}
+		log.Printf("[DEBUG] Retrying on error Refreshing Machine Config V2 %s: %v", d.Id(), err)
+		if IsNotFound(err) || IsForbidden(err) {
+			d.SetId("")
+			return fmt.Errorf("[INFO] Machine Config V2 %s not found: %s", d.Id(), err)
```

Comment:
An error prefixed with `[INFO]` is likely a copy-pasta leftover.

Reply:
`[INFO]` errs are commonly listed throughout the terraform provider. @Oats87 pointed out to me that the log message on line 62 should actually be an INFO message

What's the ask here?

Reply:
This is the only place I am seeing in the terraform provider where an error is returned with a prefix of `[INFO]`. It doesn't seem correct that an error should have such a prefix.

The ask is to remove the prefix of `[INFO]` on this error.

Reply:
Ah yes, that makes more sense. Will do